src/internal/mod.rs

```diff
@@ -0,0 +1,12 @@
+/// Internal trait to get access to the container storing the annotations.
+/// Used for the generic implementation of `AnnotationExt`
+pub trait HasAnnotations {
```

Comment:
Any objections to removing the `Has` from these traits?

Reply:
I couldn't find/google for reason to name these with HasXXX so I can remove it